skosprovider_getty/utils.py

```diff
@@ -158,7 +150,7 @@ def _create_label(literal, type):
     try:
         l = Label(literal.toPython(), type, language)
     except ValueError as e:
-        log.warn(e)
+        log.warn('Received a label with an invalid language tag: %s.' % language)
```

Comment:
The more common way would be to use a comma to give the string parameters.
Then the string doesn't get formatted when it doesn't have to (when the log level is too low)
```
log.warn('Received a label with an invalid language tag: %s.', language)
```